packages/manager/modules/user-contracts/src/translations/Messages_en_GB.json

```diff
@@ -8,5 +8,6 @@
   "user_contracts_modal_submit_button": "Continue",
   "user_contracts_modal_submit_error": "An error has occurred. Please try again.",
   "user_contracts_modal_display_text_tos": "Terms of Service",
-  "user_contracts_modal_display_text_pp": "Privacy Policy"
+  "user_contracts_modal_display_text_pp": "Privacy Policy",
+  "wizard_confirm": "Confirm"
```

Comment:
I guess we cannot add translations in other languages, except FR. Please correct me if I am wrong :)